### DISCUSSION TITLE
How can i download ?

### DISCUSSION THREAD

**Discussion by krabhi1** [hoppscotch/hoppscotch#3102] [Q&A] [answered]
How can i download ?

**Comment by liyasthomas** [comment, marked as answer]
Hoppscotch mainly has a [web client](https://docs.hoppscotch.io/documentation/clients/web) which can be installed as a PWA from the browser.

Learn how to [install Hoppscotch PWA](https://docs.hoppscotch.io/documentation/clients/web#install-hoppscotch-pwa) from Chrome browser.
